Source/WarpX.cpp

```diff
@@ -141,7 +141,7 @@ int WarpX::current_centering_noz = 2;
 bool WarpX::use_fdtd_nci_corr = false;
 bool WarpX::galerkin_interpolation = true;
 
-bool WarpX::use_filter        = false;
+bool WarpX::use_filter = true;
 bool WarpX::use_kspace_filter       = false;
```

Comment:
I would suggest changing the default value of `use_kspace_filter` to true also, otherwise it would be confusing (since it will be set to `use_filter` anyway).

Reply:
Good point, thank you. I think I had in mind the old version of the code, where we needed to be careful about having both flags set to `true`, but you fixed that already some time ago. Done in https://github.com/ECP-WarpX/WarpX/pull/2031/commits/c8e2852673306b7e7afed5ecb7156e92cf25bb3f.

Reply:
@dpgrote Please note that I reverted this change due to a runtime bug with the PSATD binary code and the FDTD runtime solver in RZ geometry. See discussion with @ax3l in the other comments, as well as the inline comment added above the defaults here. Reference commit: https://github.com/ECP-WarpX/WarpX/pull/2031/commits/39228f9eb640799f9510b3fd9422d5e6d4a78d8d.